src/zeroband/inference/parquet.py

```diff
@@ -22,10 +40,18 @@ def get_parquet_table(
         assert request_output.request_id == request_rewards.request_id
         for output, reward in zip(request_output.outputs, request_rewards.rewards):
             assert output.index == reward.completion_id
+
+            # Extract logprobs if enabled and available
+            output_logprobs = extract_logprobs(output.logprobs) if enable_logprobs else None
+            # For input logprobs, we typically don't have them from vllm, so set to None or zeros
```

Comment:
I believe there is a way to get them. However, we shouldnt need them so maybe it could be better to set that as the comment